tracing-subscriber/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 0.3.18 (November 6, 2023)
+
+This release of `tracing-subscriber` adds support for `NO_COLOR` in
+`fmt::Layer` and reintroduces support for the `chrono` crate.
```

Comment:
let's make `chrono` a link to https://crates.io/crates/chrono ?

Reply:
good idea!